charts/camunda-platform/README.md

```diff
@@ -388,25 +388,28 @@ elasticsearch:
 This chart supports the addition of Zeebe Exporters by using initContainer as shown in the following example:
 
 ```
-extraInitContainers: |
+extraInitContainers:
   - name: init-exporters-hazelcast
-    image: busybox:1.28
+    image: busybox:1.35
     command: ['/bin/sh', '-c']
-    args: ['wget --no-check-certificate https://repo1.maven.org/maven2/io/zeebe/hazelcast/zeebe-hazelcast-exporter/0.8.0-alpha1/zeebe-hazelcast-exporter-0.8.0-alpha1-jar-with-dependencies.jar -O /exporters/zeebe-hazelcast-exporter.jar; ls -al']
+    args: ['wget --no-check-certificate https://repo1.maven.org/maven2/io/zeebe/hazelcast/zeebe-hazelcast-exporter/0.8.0-alpha1/zeebe-hazelcast-exporter-0.8.0-alpha1-jar-with-dependencies.jar -O /exporters/zeebe-hazelcast-exporter.jar; ls -al exporters']
```

Comment:
:wrench: If we're explicitly putting them in `/exporters`, just in case we ever change the workdir.

```suggestion
    args: ['wget --no-check-certificate https://repo1.maven.org/maven2/io/zeebe/hazelcast/zeebe-hazelcast-exporter/0.8.0-alpha1/zeebe-hazelcast-exporter-0.8.0-alpha1-jar-with-dependencies.jar -O /exporters/zeebe-hazelcast-exporter.jar; ls -al /exporters']
```